installer.sh

```
@@ -131,8 +131,8 @@
~_pkgx_is_old() {
   v="$(/usr/local/bin/pkgx --version || echo pkgx 0)"
   /usr/local/bin/pkgx --silent semverator gt \
     $(curl -Ssf https://pkgx.sh/VERSION) \
-    $(echo $v | awk '{print $2}')
+    $(echo $v | awk '{print $2}') # FIXME
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)